NonsoAmadi10/Unblocker-Api
# Authentication

### What does this PR do?
- Creates `/api/v1/auth/register` 

### Steps Taken to Produce
- Write Tests that fulfill business logic
- Create API endpoints to pass the test

### How should this be manually Tested?
- Clone this Repo on your local machine
- Check out to this Branch
- Create a virtual environment using `pipenv shell`
- Install all dependencies using `pipenv install`
- Start the server using `flask run`
- Open postman on the above route and pass in the json body request ` { display_name: String, email: String, password: String } `
- You'd get a success message if credentials entered are correct

### Any Background Context 
- Pipenv is intalled using `pip install pipenv` command
